indico/modules/categories/templates/management/move_categories.html

```diff
@@ -1,18 +0,0 @@
-{% from 'confirmation_dialog.html' import confirmation_dialog %}
```

Comment:
Do we really want to switch to a much less verbose confirmation prompt here? I don't want people to accidentally move a category when they just wanted to move events..